### ISSUE TITLE
transferring performance on coco dataset

### ISSUE THREAD

**Issue by UcanSee** [taoyang1122/pytorch-SimSiam#7]
Hello, Have you ever try to transfer the simsiam model to object detection task. I have trained a 200-epoch model on ImageNet through simsiam（following the optimal config in paper),  Then I take it as pre-trained model of Faster-RCNN and use the transferring config of moco-v2, but I got bad performance. I wonder whether there are different training schedule when transferring self-supervised pre-trained model to object detection task? 
Hope to get your suggestiones!

**Comment by taoyang1122** [comment]
Hi, I didn't try the transfer learning experiments. But according to the paper, I think basically you can follow the MOCO codebase to do the finetuning, but you may need to tune the learning rate. BTW, how is the performance if you use MOCO-pretrained model for object detection following the same setting? Is it able to reproduce the results?
